<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettingWindow</class>
 <widget class="QWidget" name="GameSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameSettingWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="shadowWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777213</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <widget class="QPushButton" name="StartGameBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>500</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
	border: 2px solid rgb(170, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackToMenuBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>430</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
	border: 2px solid rgb(170, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelMaxCol">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Max Col</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnColUp">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/up</normaloff>:/Image/button/up</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnColDown">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/down</normaloff>:/Image/button/down</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="checkBoxSP">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe 宋体 Std L</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="statusTip">
       <string>status test</string>
      </property>
      <property name="whatsThis">
       <string>this is super potion</string>
      </property>
      <property name="text">
       <string>Super Potions</string>
      </property>
      <property name="icon">
       <iconset resource="../../snakeresource.qrc">
        <normaloff>:/Image/food/purplePotion</normaloff>:/Image/food/purplePotion</iconset>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelMaxRow">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Max Row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRowUp">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/up</normaloff>:/Image/button/up</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRowDown">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/down</normaloff>:/Image/button/down</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalWidget_3" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnWallSetLeft">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/left</normaloff>:/Image/button/left</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelWallSet">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wall Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWallSetRight">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/right</normaloff>:/Image/button/right</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalWidget_4" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnPlayerSetLeft">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/left</normaloff>:/Image/button/left</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelPlayerSet">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Player Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlayerSetRight">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/right</normaloff>:/Image/button/right</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="checkBoxHT">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe 宋体 Std L</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Harsh Terrain</string>
      </property>
      <property name="icon">
       <iconset resource="../../snakeresource.qrc">
        <normaloff>:/Image/wall</normaloff>:/Image/wall</iconset>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalWidget_5" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelGameSpeed">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Game Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpeedUp">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/up</normaloff>:/Image/button/up</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpeedDown">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover
{
	border: 4px solid rgb(170, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../snakeresource.qrc">
           <normaloff>:/Image/button/down</normaloff>:/Image/button/down</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="checkBoxTS">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe 宋体 Std L</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trash</string>
      </property>
      <property name="icon">
       <iconset resource="../../snakeresource.qrc">
        <normaloff>:/Image/food/poo</normaloff>:/Image/food/poo</iconset>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelGameSpeed_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>550</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel:hover{
color: rgb(72, 72, 218);
}</string>
      </property>
      <property name="text">
       <string>Made By KONY</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../snakeresource.qrc"/>
 </resources>
 <connections/>
</ui>
